docs/streams.md

```diff
@@ -5,91 +5,83 @@ EventStoreDB is purpose-built for event storage. Unlike traditional state-based
+filter events marked as deleted. During a stream read, the index references the stream's metadata to determine if any events have been deleted.
+
+::: note
+You cannot selectively delete events from the middle of a stream. EventStoreDB only allows to _truncate_ the stream.
+:::
+
+When you delete a stream, EventStoreDB offers two options: **soft delete** or **hard delete**.
 
-You cannot delete events from the middle of the stream. EventStoreDB only allows to _truncate_ the stream.
+**Soft delete** triggers scavenging, removing all events from the stream during the subsequent scavenging process. This allows for the reopening of the stream by appending new events. 
 
-When you delete a stream, you can use either a soft delete or hard delete. When a stream is soft-deleted, all
-events from the stream get scavenged during the next scavenging run. It means that you can reopen the stream
-by appending to it again. When using hard delete, the stream gets closed with a tombstone event. Such an event
-tells the database that the stream cannot be reopened, so any attempt to append to the hard-deleted stream
-will fail. The tombstone event doesn't get scavenged.
+**Hard delete** closes the stream with a tombstone event, signaling that the stream cannot be reopened. Any attempt to append to a hard-deleted stream will fail, and the tombstone event remains untouched.
 
-The `$all` stream bypasses the index, meaning that it does not check the metadata to determine whether events
-exist or not. As such, events that have been deleted are still be readable until a scavenge has removed them.
-There are requirements for a scavenge to successfully remove events, for more information about this, read
-the [scavenging guide](operations.md#scavenging-events).
+It's worth noting that the **`$all`** stream circumvents index checking. Deleted events within this stream remain readable until a scavenging process removes them. To understand the prerequisites for successful event removal through scavenging, refer to the [scavenging guide](operations.md#scavenging-events).
 
-EventStoreDB will always keep one event in the stream even if the stream was deleted, to indicate the stream
-existence and the last event version. Therefore, we advise you to append a specific event like `StreamDeleted`
-and then set the max count to one to keep the stream or delete the stream. Keep that in mind when deleting
-streams that contain sensitive information that you really want to remove without a trace.
+Even if a stream has been deleted, EventStoreDB retains one event within the stream to indicate the stream's existence and provide information about the last event version. As a best practice, consider appending to a specific event like **`StreamDeleted`** and then setting the **`MaxCount`** to 1 to either keep or delete the stream. Keep this in mind, especially when dealing with streams containing sensitive data that you want to erase thoroughly and without a trace.
```

Comment:
...consider appending a specific event..." 
Also, this advice is a bit unclear. Does this mean only set the `MaxCount` to 1 if you want to keep the stream, and otherwise what do you do if you want to completely erase the stream?

Reply:
Typo, it should say "to delete the stream". I'll update it now.